build.gradle

```diff
@@ -38,16 +38,3 @@ allprojects {
 dependencyGuard {
     configuration("classpath")
 }
-
-// configure SDK to use the generated_stone_source output directory so we can track changes in Git
-def sdkProject = findProject(':dropbox-sdk-java')
```

Comment:
I believe this is considered cross-project config, which is generally a discouraged practice. I moved this to the `:dropbox-sdk-java` project